configs/classification_heavy_model.yaml

```diff
@@ -1,24 +1,24 @@
-# Example configuration for training a predefined detection model
+# Example configuration for training a predefined heavy classification model
 
 model:
-  name: coco_detection
+  name: segmentation_light
   predefined_model:
-    name: DetectionModel
+    name: ClassificationModel
     params:
-      use_neck: True
+      variant: "heavy"
 
 loader:
   params:
-    dataset_name: coco_test
+    dataset_name: cifar10_test
 
 trainer:
   preprocessing:
-    train_image_size: [&height 256, &width 320]
-    keep_aspect_ratio: False
+    train_image_size: [&height 384, &width 512]
```

Comment:
We don't need to capture `height`, `width`, and `epochs` in all the yaml files. We can do it only when the captures are actually used (we're using `epochs` for `T_max` in some schedulers and `width` and `height` for mosaic augmentation)